examples/function_node/main.tf

```diff
@@ -13,15 +14,226 @@ provider "aci" {
+  sequence_number                     = "0"
+}
+
+# Create a function node with type FW_ROUTED and relation to the 3rd party firewall
+resource "aci_function_node" "function_node_fw" {
+  l4_l7_service_graph_template_dn      = aci_function_node.function_node_nlb.l4_l7_service_graph_template_dn
+  name                                 = "N1"
+  func_template_type                   = "FW_ROUTED"
+  relation_vns_rs_node_to_cloud_l_dev  = aci_cloud_l4_l7_third_party_device.third_party_fw.id
+  l4_l7_device_interface_consumer_name = "trust"
+  l4_l7_device_interface_provider_name = "untrust"
+  managed                              = "no"
+}
+
+# Create L4-L7 Service Graph connection with template T1 and the first node N0.
+resource "aci_connection" "sg1_t1-n0" {
+  l4_l7_service_graph_template_dn = aci_l4_l7_service_graph_template.sg1.id
+  name                            = "CON0"
+  adj_type                        = "L3"
+  conn_dir                        = "provider"
+  conn_type                       = "external"
+  direct_connect                  = "no"
+  unicast_route                   = "yes"
+  relation_vns_rs_abs_connection_conns = [
+    aci_l4_l7_service_graph_template.sg1.term_cons_dn,
+    aci_function_node.function_node_nlb.conn_consumer_dn
+  ]
+}
+
+# Create L4-L7 Service Graph connection with current node N0 and next node N1.
+resource "aci_connection" "sg1_n0-n1" {
+  l4_l7_service_graph_template_dn = aci_l4_l7_service_graph_template.sg1.id
+  name                            = "CON1"
+  adj_type                        = "L3"
+  conn_dir                        = "provider"
+  conn_type                       = "external"
+  direct_connect                  = "no"
+  unicast_route                   = "yes"
+  relation_vns_rs_abs_connection_conns = [
+    aci_function_node.function_node_nlb.conn_provider_dn,
+    aci_function_node.function_node_fw.conn_consumer_dn
+  ]
+}
+
+# Create L4-L7 Service Graph connection with the last node N1 and template T2.
+resource "aci_connection" "sg1_n1-t1" {
+  l4_l7_service_graph_template_dn = aci_l4_l7_service_graph_template.sg1.id
+  name                            = "CON2"
+  adj_type                        = "L3"
+  conn_dir                        = "provider"
+  conn_type                       = "external"
+  direct_connect                  = "no"
+  unicast_route                   = "yes"
+  relation_vns_rs_abs_connection_conns = [
+    aci_function_node.function_node_fw.conn_provider_dn,
+    aci_l4_l7_service_graph_template.sg1.term_prov_dn
+  ]
+}
+
+#  2. Create second L4-L7 Service Graph Template 'sg2' with type cloud
+#    Create two nodes with additional parameters
+#     - The first node 'N0' with type ADC_ONE_ARM and relation to the cloud native load balancer
+#     - The second node 'N1' with type FW_ROUTED and relation to the 3rd party firewall
+#    Create the connection between the templates ('T1 - consumer' and 'T2 - provider') and nodes ('N0' and 'N1').
+
```

Comment:
What is the difference between SG1 and SG2?

Reply:
Was this addressed?

Reply:
yes and changed